### PR TITLE
Make images customizable (with instructions)

### PR DESCRIPTION
**Pull request by DjebbZ**
Here you go. Tested successfully in my local Firefox.

### CODE REVIEW

**Comment by DjebbZ** [comment]
See #7

File: index.html [hunk 1]
```diff
@@ -178,6 +178,7 @@
         </div>
       </footer>
     </div>
+    <script type="text/javascript" src="images.js"></script>
```
Comment (MoOx) [inline index.html]: type="text/javascript"  is useless :D

**Comment by MoOx** [comment]
This is breaking the online version
.sample should be force added as images.js

**Comment by kud** [comment]
Yep.

**Comment by DjebbZ** [comment]
I didn't know about `add --force`, hope it will do the trick.

**Comment by MoOx** [comment]
You can remove the .sample & adjust the README ;)

**Comment by DjebbZ** [comment]
Yep

**Comment by DjebbZ** [comment]
If I remove the sample, will people still have the original images.js to play with ?

Yes, but editing images.js is visible by git, even if in .gitignore (I deleted and re-downloaded my fork to test).

**Comment by MoOx** [comment]
Well done. Thanks !

**Comment by MoOx** [comment]
Oh.

**Comment by MoOx** [comment]
Weird.

**Comment by DjebbZ** [comment]
Indeed.

**Comment by DjebbZ** [comment]
Maybe, reverting to previous commit, and manually add the really ignored `images.js` file will do the trick ?

**Comment by MoOx** [comment]
online version need the file to be versionned (for now).
Not sure what is the best way to handle that.
Anyway, it's a step. I would like to drop this manual file for the long run (see  https://github.com/putaindecode/pretty-new-tab/issues/7#issuecomment-57624906)

**Comment by DjebbZ** [comment]
Sorry for the weird situation I put you in...

**Comment by MoOx** [comment]
Not a big deal.

**Comment by kud** [comment]
The best way would be to have master which compiles to gh-pages and add images.js in it.

**Comment by DjebbZ** [comment]
Nice idea ! A post-commit hook ?

**Comment by kud** [comment]
or gulp-gh-pages.

**Comment by DjebbZ** [comment]
Can I leave this stuff to you guys ?

**Comment by kud** [comment]
Yesh.

**Comment by MoOx** [comment]
Yeah we will end up that way: clean decoupled code
Here is a issue to discuss about that

https://github.com/putaindecode/pretty-new-tab/issues/23

**Comment by DjebbZ** [comment]
Thanks ! I need to get back to work (you know, the "real world" :/ :D)

**Pull request merged**